Dockerfile

```diff
@@ -9,6 +9,7 @@ FROM ruby:2.5.0
 #
 ARG RAILS_ENV=production
 ENV RAILS_ENV ${RAILS_ENV}
+ENV DEVICE_SECRET_KEY='changeme'
```

Comment:
@yssavchuk Could you please add comment why this is needed: `Devise requires secret key to be set or it raises an error preventing from running any scripts. Users should override this variable by passing environment variable to Docker`.